react/components/FilterBar/Menu.js

```diff
@@ -169,6 +212,10 @@ Menu.propTypes = {
   onClose: PropTypes.func,
   /** Menu Box align (default is right) */
   align: PropTypes.oneOf(['right', 'left']),
+  /** Function to handle callback on overlay click */
+  onBackgroundClick: PropTypes.func,
+  /** Device in use returned byt the useDeviceHook  */
+  device: PropTypes.string,
 }
 
-export default Menu
+export default useDeviceHook(Menu)
```

Comment:
as I said in the file, this should be renamed because it is not a hook but a hoc.